src/components/shared/DiagramCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Card, CardProps } from "antd";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+import { IGatsbyImageData } from "gatsby-plugin-image";
+
+const {
+    diagramCard,
+    diagramCardFooter,
+} = require("../../style/disease-subpage.module.css");
+
+export interface Diagram {
+    image: {
+        childImageSharp: {
+            gatsbyImageData: IGatsbyImageData;
+        };
+    };
+    title: string;
+    caption: string;
+}
```

Comment:
sorry, I was confused before about this interface, I didn't realize it was also the query result, so my comments probably were confusing. my only suggestion other than just importing the interface Diagram from component-queries/types.ts is you could adjust the data so that image you send into the component is just the iGatsbyImageData, since there is no need for it to be nested anymore.